libs/execution_context/ebpf_program.c

```diff
@@ -53,9 +53,12 @@ typedef struct _ebpf_program
     const ebpf_extension_data_t* info_extension_provider_data;
     // Program type specific helper function count.
     uint32_t provider_helper_function_count;
+    // Global helper function count implemented by the extension.
+    uint32_t global_helper_function_count;
     bool invalidated;
 
     ebpf_trampoline_table_t* trampoline_table;
+    ebpf_trampoline_table_t* global_helper_trampoline_table;
 
     // Array of helper function ids referred by this program.
```

Comment:
Does this array include both global and program type specific helper function ids?

Reply:
Yes.